spec/_attachments/ic.did

```diff
@@ -131,18 +131,36 @@ type bitcoin_get_balance_args = record {
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_result = satoshi;
+
 type bitcoin_get_balance_query_args = record {
     address : bitcoin_address;
     network : bitcoin_network;
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_query_result = satoshi;
+
+type bitcoin_get_current_fee_percentiles_args = record {
+    network : bitcoin_network;
+};
+
+type bitcoin_get_current_fee_percentiles_result = vec millisatoshi_per_byte;
+
 type bitcoin_send_transaction_args = record {
     transaction : blob;
     network : bitcoin_network;
 };
 
-type millisatoshi_per_byte = nat64;
+type bitcoin_get_block_headers_args = record {
+	start_height : nat32;
```

Comment:
nit: we're using `nat32` for height in a number of places. Would it make sense to define this as a type? (e.g. `type bitcoin_block_height = nat32`)

Reply:
Good idea! I added the type definition. I also changed `block_hash` to `bitcoin_block_hash` for the sake of consistency.
Do you agree with this change?